validators/redirect_uri_validator.py

```
@@ -0,0 +1,75 @@
+            if not parsed.scheme:
+                return "invalid_redirect_uri", "Redirect URI must be a valid URL"
+
+            # Check scheme (must be http or https)
+            if parsed.scheme not in ('http', 'https'):
+                return "invalid_redirect_uri", "Redirect URI must use HTTPS (or HTTP for localhost)"
+
+            # Check for valid netloc (host) - must exist for http/https
+            if not parsed.netloc:
+                return "invalid_redirect_uri", "Redirect URI must be a valid URL"
+
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)